docs/MS Excel/Границы ячеек.md

```diff
@@ -0,0 +1,28 @@
+≡ Краткое руководство Markdown
+
+# Границы ячеек
+
+**Рисование границ**
+
+Чтобы настроить тип линии или цвет границ ячеек или удалить существующие границы, используйте параметры «Нарисовать границы». Чтобы нарисовать границы ячеек, сначала выберите тип границы, затем цвет и тип линии, а затем выберем ячейки, вокруг которых нужно добавить границу. Вот как это сделать:
+  1. На вкладке «Главная» рядом с элементом «Границы» щелкните стрелку
+  2. Для создания внешних границ выберите пункт «Нарисовать границы», для внутренней разметки — пункт «Нарисовать линии границы».
+  3. Щелкните стрелку рядом с элементом «Границы2, выберите пункт «Цвет линии» и укажите нужный цвет.
+  4. Щелкните стрелку рядом с элементом «Границы», выберите пункт «Вид линии» и укажите нужный вид.
+  5. Выделите ячейки, вокруг которых необходимо нарисовать границы.
+ 
+**Объединение ячеек** 
+
+Чтобы объединить ячейки нужно:
+  1. Выделите ячейки для объединения.
+  2. Выберите «Объединить и поместить в центре».
+
+
+
+
+---
+
+###### Links:
```

Comment:
[ch-md-error-increment-heading-levels](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-increment-heading-levels)